Add more content to wallet migration guide [WIP]

RichardIrala opened the pull request.

JamesLawton commented on docs/pages/api/indexer/examples/webhook-listener.mdx:
```diff
@@ -1,46 +1,79 @@
+You can listen to transactions via webhooks using the Sequence Indexer by registering an endpoint or removing it after it's no longer required.
+
+<Callout type='warning'>
+	If you prefer a no-code way to add webhooks that uses the Sequence Builder, check out [this walkthrough]()
```
Link needs to be added as it is currently blank.

JamesLawton commented on docs/pages/api/indexer/examples/webhook-listener.mdx:
```diff
@@ -1,46 +1,79 @@
+2. [Remove a Webhook Listener](/api/indexer/examples/webhook-listener#2-remove-a-webhook-listener)
+
+Examples
+1. [Listen to All Mints of a Contract](/api/indexer/examples/webhook-listener#1-listen-to-all-mints-of-a-contract)
```
This link doesn't work and should be updated to `/api/indexer/examples/webhook-listener#1-listen-to-all-mints-of-a-collectible-contract`

JamesLawton commented on docs/pages/api/indexer/examples/webhook-listener.mdx:
```diff
@@ -1,46 +1,79 @@
+<Callout type="info">
+	If you require a server for development, you can use one of the following:
+
+	[Template Nodejs Webhook Server](https://github.com/0xsequence-demos/template-nodejs-webhook-server/blob/master/README.md) combined with [ngrok](https://ngrok.com) a secure tunnel to your computer running the local server
```
Templates are awesome, love em!

JamesLawton commented on docs/pages/api/indexer/examples/webhook-listener.mdx:
```diff
@@ -78,7 +111,216 @@ func GetTransactionHistory(accountAddress string) {
+
+### 2. Listen to a Specific Token ID
+
+If you want to get all mints of a specific token ID
```
add in the description 'For an ERC1155' - that way it's clear why you would want to monitor a specify token ID if it's semi-fungible.

JamesLawton commented on docs/pages/api/indexer/examples/webhook-listener.mdx:
```diff
@@ -78,7 +111,216 @@ func GetTransactionHistory(accountAddress string) {
 
-Now you can listen to events on the blockchain with your webhook.
\ No newline at end of file
+### 4. Listen to Marketplace Request Creation
```
Let's add a brief sentence describing the utility, i.e., 'For example, you can track new requests on your custom marketplace and notify an enduser of the update'.

JamesLawton commented on docs/pages/solutions/builder/getting-started.mdx:
```diff
@@ -8,13 +9,13 @@ Let's get started!
 ## Sign up for Sequence Builder
 
-### Step 1: Go to sequence.build
+### 1: Go to sequence.build
```
When we do sequential steps in the docs, let's use the vocs markdown for steps as I find it much cleaner and more legible. Here's the markdown reference: https://vocs.dev/docs/markdown#steps

RichardIrala committed:
Add migration guide from ThirdWeb to Sequence and improve JIT Migration process guide

RichardIrala committed:
Merge dc92ff0af8987975928ebe937ff191a567421264 into 1935062f74d8a373de39f370bf6ab19b93329162

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 113/merge

JamesLawton committed:
updated sequence diagram and text

RichardIrala committed:
Merge 4284718e822adb57541b44452f26980714b1fef1 into 1935062f74d8a373de39f370bf6ab19b93329162

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 113/merge

JamesLawton committed:
updated migration guide

JamesLawton committed:
updated migration guide

RichardIrala committed:
Merge 6c5bd1ccf37220bd0c18da6f0ca96dcef43df30e into 1935062f74d8a373de39f370bf6ab19b93329162

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 113/merge

andygruening committed:
Guide: Epic Auth in Unreal (#114)

* started epic auth in unreal guide

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 114/merge

* further changes to epic auth unreal guide

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 114/merge

* some changes

* chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 114/merge

---------

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

RichardIrala committed:
Add quotes to the title in migration.mdx

RichardIrala committed:
Merge 34e27639ee404d5d69f8a61435cedff7988d7a3b into a9d46095699f5d4073218d8908e24041157f1535

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: 113/merge
